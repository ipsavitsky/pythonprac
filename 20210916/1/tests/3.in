123, ["a", "b", "c"], 321
