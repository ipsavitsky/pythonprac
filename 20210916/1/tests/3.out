<class 'int'> <class 'list'> <class 'int'>
